libc/src/string/string_utils.h

```diff
@@ -79,7 +79,8 @@ LIBC_INLINE size_t string_length_wide_read(const char *src) {
   return char_ptr - src;
 }
 
-LIBC_INLINE size_t string_length_byte_read(const char *src) {
+template<typename T>
+LIBC_INLINE size_t string_length_trivial(const T *src) {
```

Comment:
help wanted renaming this. Also, should I use cpp::is_same for SFINAE for char and wchar_t?

Reply:
4f34e6452da3 PTAL